client/components/info-tooltip/index.js

```diff
@@ -45,21 +46,23 @@ export default class InfoTooltip extends Component {
 	}
 
 	render() {
+		const anchor = this.props.anchor || <Gridicon icon="info-outline" size={ 18 } />;
+
 		return (
 			<span
 				onMouseEnter={ this.openTooltip }
 				onMouseLeave={ this.closeTooltip }
 				className={ classNames( 'info-tooltip', this.props.className ) } >
-				<Gridicon ref="icon" icon="info-outline" size={ 18 } />
+				<span ref="anchor">{ anchor }</span>
 				{ this.state.showTooltip &&
 					<Tooltip
-						className="wc-connect-popover"
+						className="info-tooltip__container wcc-root"
```

Comment:
Because the tooltip is created in a different `<div>` separated of the rest of the React UI (think of it as a mini-Modal). So we need the root rule to apply styles to it.